redfish-core/lib/license_service.hpp

```diff
@@ -265,6 +265,46 @@ inline void requestRoutesLicenseEntryCollection(App& app)
         });
 }
 
+inline std::string
+    translateLicenseTypeDbusToRedfish(const std::string& licenseType)
+{
+    if (licenseType == "com.ibm.License.Entry.LicenseEntry.Type.Purchased")
+    {
+        return "Production";
+    }
+    if (licenseType == "com.ibm.License.Entry.LicenseEntry.Type.Prototype")
+    {
+        return "Prototype";
+    }
+    if (licenseType == "com.ibm.License.Entry.LicenseEntry.Type.Trial")
+    {
+        return "Trial";
+    }
+    return "";
```

Comment:
This would be a validator error if this is empty 
LicenseType="" would be a validator error 

else { 
// Any other values would be invalid 
BMCWEB_LOG_DEBUG << "PowerMode value was not valid: " << modeValue; 
messages::internalError(aResp->res); 
}

Reply:
it won't be error if value is empty, validator just throws a warning which can be ignored

Reply:
Are you sure? I thought since it was an enum, it would throw an error if empty

Reply:
Another choice would be 
             std::string licenseType = translateLicenseTypeDbusToRedfish(*value);
             if (!licenseType .empty())
             
             like 
             https://github.com/ibm-openbmc/bmcweb/pull/181/files

Reply:
yeah I observed only warning for empty string with validator.
anyways I have modified as you suggested.
please have a look